bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/ThrottlingClientInterceptor.java

```diff
@@ -98,14 +129,14 @@ public void onClose(io.grpc.Status status, Metadata trailers) {
                 delegate().onClose(status, trailers);
               }
             };
-        delegate().start(markCompletionListener, headers);
-        delegate().request(numMessagesRequested);
-        delegate().sendMessage(message);
+        delegateCall.start(markCompletionListener, headers);
+        delegateCall.request(numMessagesRequested);
+        delegateCall.sendMessage(message);
       }
 
       @Override
-      protected ClientCall<ReqT, RespT> delegate() {
-        return delegateCall;
+      public void halfClose() {
```

Comment:
Should we propgate the `halfClose()`, or is it irrelevant for our use case?

Reply:
I just put the halfClose as part of the sendMessage since its a unary call